portfolio/src/main/webapp/index.html

```diff
@@ -2,18 +2,19 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Santiago's Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
+      <h1>Santiago's Portfolio</h1>
+      <img id="profile" src="/images/profile.jpg"/>
```

Comment:
nit: Does this work as is?  I think it might need to changed from:
/images/profile.jpg -> images/profile.jpb

Reply:
Fixed!